kmq.js

```diff
@@ -14,31 +15,78 @@ const db = new sqlite3.Database('./main.db', (err) => {
 var scoreboard = {};
 var currentSong = null;
 var currentArtist = null;
+var currentSongLink = null;
 var gameInSession = false;
+
+const sendSongMessage = (message, currentSong, currentArtist, currentSongLink, isQuit) => {
```

Comment:
`currentSong`, `currentArtist`, and `currentSongLink` are already global and accessible from within this function